examples/src/world-to-screen.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import CameraControls from 'camera-controls';
+import * as reveal from '@cognite/reveal';
+import { CogniteClient } from '@cognite/sdk';
+import { CadNode } from '@cognite/reveal/threejs';
+import { Vector3 } from 'three';
+import { GUI } from 'dat.gui';
+
+CameraControls.install({ THREE });
+
+// Want to split the logic away from THREE at some point.
+function project3DPositionTo2DPlane(
+  canvas: HTMLCanvasElement,
+  camera: THREE.PerspectiveCamera,
+  position3D: THREE.Vector3
+): { x: number; y: number } {
+  const vector = position3D.clone();
+  vector.project(camera);
+  vector.x = Math.round((0.5 + vector.x / 2) * (canvas.width / window.devicePixelRatio));
+  vector.y = Math.round((0.5 - vector.y / 2) * (canvas.height / window.devicePixelRatio));
+
+  return { x: vector.x, y: vector.y };
+}
```

Comment:
Move to `CadNode` or standalone utility function. Can we unit test this? #coronachallenge

Reply:
Rename to `getScreenPosition(worldPosition)` or something